config/main.py

```diff
@@ -33,6 +35,64 @@ def run_command(command, display_cmd=False, ignore_error=False):
     if proc.returncode != 0 and not ignore_error:
         sys.exit(proc.returncode)
 
+def convert_interface_name(interface_name):
```

Comment:
This function will blindly convert a port name to a port alias and vice-versa. This could pose issues if a device shares the same string for both a port name and a port alias, as it will always find a match on line 49 and return that port's alias.

I think it is better to have two functions with distinct functionality: `interface_name_to_alias(interface_name)` and `interface_alias_to_name(interface_alias)`.

Reply:
Introduced interface_alias_to_name() to convert the alias to default mode. In the current code change, there is no any need for interface_name_to_alias() conversion. While implementing the feature for other subset of commands if needed will implement on need basis.